use_cases/eluc/persistence/persistors/persistor.py

```diff
@@ -0,0 +1,26 @@
+from pathlib import Path
+
+from abc import ABC, abstractmethod
+
+from persistence.serializers.serializer import Serializer
+
+class Persistor(ABC):
+    """
+    Abstract class for persistors to inherit from.
+    """
+    def __init__(self, file_serializer: Serializer):
+        self.file_serializer = file_serializer
```

Comment:
I was actually expecting a `FileSerializer` class, maybe an interface, implemented by the other ones.
And I was thinking `HuggingFacePersistor` can only accept `FileSerializer` instances.

Reply:
`Serializer` is an interface that is implemented by `NeuralNetworkSerializer`, `SKLearnSerializer`, etc. It has methods `save` and `load`. The `Persistor` abstract class must wrap around a `Serializer`.